README.md

```diff
@@ -1,2 +1,30 @@
 # diffusion-slice-rs
+
 Take screenshots of 3D and 4D images
+
+This project aims to
+
+- offer a simple cmdline application
+- build a [wgpu](https://github.com/gfx-rs/wgpu) 3D scene from
+  - a slice of a NIfTI image (`.nii; .nii.gz`)
```

Comment:
Can you remind me if we want to load the whole 3D image on the GPU or only a slice of it?

Reply:
Some images are quite heavy (like a 100x100x100x32 f32 DWI is 128Mb). Loading the whole image on GPU, knowing that we only want some slices of it seems wasteful to me. I'm not a GPU programmer, but I guess it's faster loading 100x100 f32 N times than fully loading it once then choosing which slice to display. Maybe you have a different opinion.